src/main/scala/com/fulcrumgenomics/umi/ExtractUmisFromBam.scala

```diff
@@ -113,14 +114,14 @@ class ExtractUmisFromBam
         // verify everything is in order
         Seq(r1, r2).foreach(r => {
           if (!r.getReadPairedFlag) throw new IllegalStateException(s"Read ${r.getReadName} was not paired")
-          if (!r1.getReadUnmappedFlag) throw new IllegalStateException(s"Read ${r.getReadName} was not mapped")
+          if (!r1.getReadUnmappedFlag) throw new IllegalStateException(s"Read ${r.getReadName} was not unmapped")
```

Comment:
Also, if you care, you could use `fail(msg)` instead of throwing here, and it will result in a message on the console and exiting, without a big stack trace getting printed.